base.sh

```diff
@@ -69,6 +69,7 @@ export_custom_envs() {
   export VALIDATOR_EJECTOR_LOCATOR_ADDRESS=${VALIDATOR_EJECTOR_LOCATOR_ADDRESS}
   export VALIDATOR_EJECTOR_STAKING_MODULE_ID=${VALIDATOR_EJECTOR_STAKING_MODULE_ID}
   export VALIDATOR_EJECTOR_OPERATOR_ID=${VALIDATOR_EJECTOR_OPERATOR_ID}
+  export VALIDATOR_EJECTOR__MESSAGES_PASSWORD=${VALIDATOR_EJECTOR__MESSAGES_PASSWORD}
```

Comment:
Could you replace the double underscore on the single? To match the rest style of the variables writing.
VALIDATOR_EJECTOR__MESSAGES_PASSWORD -> VALIDATOR_EJECTOR_MESSAGES_PASSWORD